sql/plan/subqueryalias.go

```diff
@@ -76,6 +76,11 @@ func (n *SubqueryAlias) WithChildren(children ...sql.Node) (sql.Node, error) {
 	return &nn, nil
 }
 
+func (n SubqueryAlias) WithName(name string) *SubqueryAlias {
+	n.name = name
```

Comment:
This is a neat trick (struct receiver to automatically return a copy), gonna use that!